iceprod/server/data/www/rest.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Helper functions that use REST API
+ */
+
+async function fetch_json(url, passkey, json=null, post=false) {
+    try {
+        var payload = {
+                headers: new Headers({
+                                'Authorization': 'Bearer ' + passkey,
```

Comment:
What happens if I don't supply a passkey, or the passkey is null?  It won't throw an error here, so maybe that should be explicitly checked for.

In general, if we can do an easy check before sending the request to the server, we should.

Reply:
2db7727
```if (passkey == null) {
        throw "fetch_json: passkey can't be null";
    }```